cypress/e2e/sessions.cy.ts

```diff
@@ -22,6 +22,8 @@
 import { User } from '../../dist'
 
 describe('Login and logout', function() {
+	before(() => cy.logout())
```

Comment:
I would prefer a try/catch or error catch somewhere in case this test is restarted. If we always logout before running this entire test (which is really checking the default state of cypress and nextcloud) does not make sense anymore. :)

Reply:
Maybe do that on fail ?
https://docs.cypress.io/api/events/catalog-of-events#Debug-the-moment-a-test-fails

Reply:
This is just run once before, so it just ensures a consistent test environment. So it ensures it does not matter if `users.cy.ts` is run first or this one.

I think we want to test the login works, so we assume we are logged out on the first test `Login and see the default files list` so we can enforce this initial state to properly test this scenario, or am I missing something?

Reply:
No, the issue is about the logic.
With independent sessions for each test, we should not have to logout. Manually adding this will circumvent the actual behaviour of those cypress tests :) 

In any case, please open a dedicated PR for those changes.
They are not related to the docker integration :pray:

Reply:
> No, the issue is about the logic. With independent sessions for each test, we should not have to logout. Manually adding this will circumvent the actual behaviour of those cypress tests :)
> 
> In any case, please open a dedicated PR for those changes. They are not related to the docker integration :pray:

I did it for you and rebase this.
See https://github.com/nextcloud/nextcloud-cypress/pull/168